### PR TITLE
Two quick fixes for minifollowups

### LINKED ISSUE #2728: Minifollowup single info should not allow incompatible trigger choices
At present if `--n-loudest` and `--trigger-id` are both passed to pycbc_page_snglinfo, `--n-loudest` is silently ignored.  Thus the value of n-loudest could be complete nonsense and in general contradictory to trigger id.  We should not allow 'nonsense' arguments that are ignored due to being redundant with another argument, so make these mutually exclusive.  (This may have other implications for minifollowup code.)

### PR DESCRIPTION
**Pull request by spxiwh**
This patch makes two quick fixes for sngl minifollowups:

 * Order the sngls in the right order again (loudest event first, not last)
 * Stop sending n-loudest by sngl_page_info, and make the code fail if it is sent both options (fixes #2728)

This is not yet tested.

### CODE REVIEW

File: bin/minifollowups/pycbc_page_snglinfo [hunk 1]
```diff
@@ -53,6 +53,11 @@ parser.add_argument('--ranking-statistic', default='newsnr',
          "'newsnr'")
 
 args = parser.parse_args()
+
+if args.trigger_id is not None and args.n_loudest is not None:
```
Comment (tdent) [inline bin/minifollowups/pycbc_page_snglinfo]: how about using mutually_exclusive_group or whatever it's called?

File: pycbc/workflow/minifollowups.py [hunk 2]
```diff
@@ -560,8 +560,6 @@ def make_sngl_ifo(workflow, sngl_file, bank_file, trigger_id, out_dir, ifo,
     node.add_input_opt('--single-trigger-file', sngl_file)
     node.add_input_opt('--bank-file', bank_file)
     node.add_opt('--trigger-id', str(trigger_id))
-    if rank is not None:
```
Comment (tdent) [inline pycbc/workflow/minifollowups.py]: surely if you remove the rank=None default value the code will say "what's this object called 'rank'" here?

Comment (spxiwh) [reply pycbc/workflow/minifollowups.py]: This line is also being removed (did I miss something??)

Comment (tdent) [reply pycbc/workflow/minifollowups.py]: oh, duh. so, no more rank for anything.

File: bin/minifollowups/pycbc_page_snglinfo (hunk 1 above)
Comment (spxiwh) [reply bin/minifollowups/pycbc_page_snglinfo]: Done

**Comment by tdent** [comment]
Fixes #2728.

**Comment by tdent** [comment]
Are we OK to merge this Ian?

**Comment by spxiwh** [comment]
This has been tested now. Merging.

**Pull request merged**